src/Blueprint/Blueprint/help/New-AzBlueprintArtifact.md

```diff
@@ -0,0 +1,373 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Blueprint.dll-Help.xml
+Module Name: Az.Blueprint
+online version: https://docs.microsoft.com/en-us/powershell/module/az.blueprint/new-azblueprintartifact
+schema: 2.0.0
+---
+
+
+# New-AzBlueprintArtifact
+
+## SYNOPSIS
+Create a new artifact and save it within a blueprint.
+
+## SYNTAX
+
+### CreateTemplateArtifact (Default)
+```
+New-AzBlueprintArtifact -Name <String> -Type <PSArtifactKind> -Blueprint <PSBlueprintBase>
+ [-Description <String>] [-DependsOn <System.Collections.Generic.List`1[System.String]>]
+ -TemplateParameterFile <String> -TemplateFile <String> [-ResourceGroupName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### CreateArtifactByInputFile
+```
+New-AzBlueprintArtifact -Name <String> -Blueprint <PSBlueprintBase> -ArtifactFile <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### CreateRoleAssignmentArtifact
+```
+New-AzBlueprintArtifact -Name <String> -Type <PSArtifactKind> -Blueprint <PSBlueprintBase>
+ [-Description <String>] [-DependsOn <System.Collections.Generic.List`1[System.String]>]
+ -RoleDefinitionId <String> -RoleDefinitionPrincipalId <String[]> [-ResourceGroupName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### CreatePolicyArtifact
+```
+New-AzBlueprintArtifact -Name <String> -Type <PSArtifactKind> -Blueprint <PSBlueprintBase>
+ [-Description <String>] [-DependsOn <System.Collections.Generic.List`1[System.String]>]
+ -PolicyDefinitionId <String> -PolicyDefinitionParameter <Hashtable> [-ResourceGroupName <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create a new template artifact. There are two ways to create an artifact: either through an artifact JSON as an input file or through providing inline parameters for the artifact. 
```

Comment:
"Create a new Azure Resource Manager template artifact. There are two ways to create an Azure Resource Manager template artifact: ..."